lighthouse-core/audits/seo/anchor-text.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('../audit');
+const URL = require('../../lib/url-shim');
+const BLOCKLIST = new Set([
+  'click here',
+  'click this',
+  'go',
+  'here',
+  'this',
+  'start',
+  'right here',
+  'more',
+  'learn more',
+]);
+
+class AnchorText extends Audit {
+  /**
+   * @return {!AuditMeta}
+   */
+  static get meta() {
+    return {
+      category: 'Content Best Practices',
+      name: 'anchor-text',
+      description: 'Anchors have descriptive text.',
```

Comment:
> Links have descriptive text

cc @rviscomi you okay with calling these "links" instead of 'anchors' ?  IMO its more natural and i dont think the technical correctness is worth it.

Reply:
"links" or "<a> elements" SGTM

Reply:
Changing `name` kinda forced me to get rid of 'anchor' completely: https://github.com/GoogleChrome/lighthouse/pull/3490/commits/b710137ed97224c1b512091864b7b8763047a357 . @paulirish PTAL